### ISSUE TITLE
Disable - WARNING: Editor operations are not supported in temporary consoles.

### ISSUE THREAD

**Issue by HerBenSte** [PowerShell/vscode-powershell#2525]
**I'm using:**
"powershell.debugging.createTemporaryIntegratedConsole": true
to always start a new console window to prevent any old code loading issues which I had in the past.
Only restart of VSCode was fixing it.
Since I use temporary consoles I never ran into that issue again.

**But now (I guess since the last update) I get this warning:**
[See screenshot](https://i.imgur.com/SBg0SvD.png)

This WARNING: comes from the
Clear-Host
in all of my scripts. Since I need those Clear-Host at the start of my Write-Host messages to get a clean console output I don't want to remove them (even not temporarly) out of my scripts.

**Is it possible to disable this warning message?**

**Comment by TylerLeonhardt** [comment]
@HerBenSte do you use this setting?:

`"powershell.integratedConsole.forceClearScrollbackBuffer": true`

**Comment by HerBenSte** [comment]
@TylerLeonhardt
No I don't but I tested it right now.

**I basically use:**
"powershell.debugging.createTemporaryIntegratedConsole": true
To make sure I always have a clean script load.
That's working nice so far. But the last update of the PowerShell-Plugin shows this yellow Warning message everytime.

I mean, I can ignore it but if there is a way to disable this message it would be appreciated! :-)

**Comment by TylerLeonhardt** [comment]
Sorry, I mean in your user settings. Open your preferences and uncheck this box:

![image](https://user-images.githubusercontent.com/2644648/76243193-2e4e8100-61f5-11ea-96b5-e1fcae5cc524.png)

**Comment by HerBenSte** [comment]
@TylerLeonhardt
This setting is not set at all in the settings.json even in no settings the checkbox is set (workspace and so on).

**I only set the value:**
"powershell.debugging.createTemporaryIntegratedConsole": true

To make sure everytime I load the PowerShell it's new and no variable are filled with something else before. But since the last update of that extension it takes like 5sec until it starts.
Before it was like 1-2sec and that wasn't really distracting.
And the WARNING message is a bit annoying aswell.

At the moment I went back to normal console and not Temporary one.
Until I realise that my PowerShell deliveres wrong stuff so that I have to open a new console. :-\
